src/index.ts

```diff
@@ -4,3 +4,4 @@ export * from "./types";
 export * from "./logger";
 export * from "./client";
 export { WorkflowError, WorkflowAbort } from "./error";
+export { resend, openai, anthropic, custom } from "@upstash/qstash";
```

Comment:
When these are exported both from qstash and workflow, would it cause a conflict of imports?

Reply:
removed the exports alltogether as they aren't needed.